### ISSUE TITLE
How to fit in my own dataset

### ISSUE THREAD

**Issue by momo1986** [chrispolo/Keypoints-of-humanpose-with-Mask-R-CNN#11]
Hello Cheispolo.

I have modified your  script on my own dataset. Generally, the configuration is like your previous Keras file. Thanks for sharing!
The keypoint number is 11:
However, I met some problems:
![image](https://user-images.githubusercontent.com/34225874/56877572-8a972f00-6a81-11e9-800e-02f0882b4e6d.png)
The loss of both training dataset and vlidation daset cannot be reduced.

How can I resolve the problem with hyper-parameter adjusting?

My input data is 256*256 pixel.

**Comment by fastovetsilya** [comment]
Try making changes in compile() function in MaskRCNN class in model.py. You can try using Adam optimizer and see if it works.
